pkg/chart/dependency_test.go

```diff
@@ -163,3 +221,51 @@ func TestFindDepByName(t *testing.T) {
 		t.Errorf("wrong dependency, got: %s , want: %s", dep.Version, "4.5.6")
 	}
 }
+
+func TestLockFilePath(t *testing.T) {
+	tests := map[string]struct {
+		chartPath     string
+		apiVersion    string
+		expectedPath  string
+		shouldFail    bool
+		expectedError error
+	}{
+		"api v1 chart": {
+			"/tmp/kafka",
+			APIV1,
+			"/tmp/kafka/requirements.lock",
+			false,
+			nil,
+		},
+		"api v2 chart": {
+			"/tmp/kafka",
+			APIV2,
+			"/tmp/kafka/Chart.lock",
+			false,
+			nil,
+		},
+		"unexisting api chart": {
+			"/tmp/kafka",
+			"vvv000",
+			"",
+			true,
+			errors.New("unrecognised apiVersion \"vvv000\""),
+		},
+	}
+
+	for name, tc := range tests {
+		t.Run(name, func(t *testing.T) {
+			want := tc.expectedPath
+			got, err := lockFilePath(tc.chartPath, tc.apiVersion)
+			if tc.shouldFail {
+				if !assert.EqualError(t, tc.expectedError, err.Error()) {
```

Comment:
why do you need `assert`?

Reply:
I don't really need it. I can compare errors strings:

~~~go
err.Error() != tc.expectedError.Error() {
...
}
~~~